views/choose_password

```diff
@@ -156,12 +176,16 @@
         while (pair = tokenize.exec(querystring))
            urlParams[re_space(pair[1])] = re_space(pair[2]);
     })();
-
+    
     var id = urlParams['id'];
     var base = PARSE_SERVER_URL;
     document.getElementById('form').setAttribute('action', base + '/apps/' + id + '/request_password_reset');
     document.getElementById('username').value = urlParams['username'];
     document.getElementById('username_label').appendChild(document.createTextNode(urlParams['username']));
+    document.getElementById("password").oninput = validatePassword;
```

Comment:
@natanrolnik not sure about that, should we use event listeners instead or is is valid?